markdownify/__init__.py

```diff
@@ -237,6 +237,24 @@ def convert_img(self, el, text, convert_as_inline):
 
         return '![%s](%s%s)' % (alt, src, title_part)
 
+    def convert_table(self, el, text, convert_as_inline):
+        rows = el.find_all('tr')
+        text_data = []
+        for row in rows:
+            headers = row.find_all('th')
+            columns = row.find_all('td')
+            if len(headers) > 0:
+                headers = [head.text.strip() for head in headers]
+                headers = [head for head in headers if head]
```

Comment:
Thank you very much for your contribution! Merging it right away, by why do you filter out the th/td that dont have content? Lets say, the upper left cell is intentionally left blank, like here:

|   | a |
|---|---|
| b | c |

Reply:
I was trying to avoid using cases where there's no content. But yes that's a good example/case. I'll make the change 👌

Reply:
Actually it seems the markdown isn't supporting empty headers, it will support empty columns but not headers. At least in my VS Code markdown preview and in an online preview. 
So I guess we either add the validation of a default string, like a '.' or wtv.

Reply:
Hey, sorry, got caught up :/

Ok, maybe we get away with replacing empty cells with `&nbsp;`? That won't show but will still be there :+1: